<!doctype html>
<html lang="en-US">
<head>
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1">
<title>Chen Lab @ GIS</title>
<link href="css/singlePageTemplate.css" rel="stylesheet" type="text/css">
</head>
<body>
<!-- Main Container -->
<div class="container"> 
  <!-- Navigation -->
  <header> <a href="">
    <h4 class="logo">CHEN LAB</h4>
    </a>
    <nav>
      <ul>
        <li><a href="#about">Research</a></li>
        <li><a href="#people">People</a></li>
		<li><a href="#publications">Publications</a></li>
		<li><a href="https://github.com/khchenlab/">Software</a></li>
		<li><a href="https://twitter.com/khchenlab">Twitter</a></li>  
        <li><a href="#contact">Contact</a></li>
      </ul>
    </nav>
  </header>
  <!-- Hero Section -->
  <section class="hero" id="hero">
    <h2 class="hero_header">Welcome to the Chen Lab at the <br>
      Genome Institute of Singapore</h2>
</section>
  <!-- About Section -->
  <section class="about">
    <h2 class="hidden">About</h2>
    <p class="text_column">We are a group of researchers at the Genome Insitute of Singapore interested in developing and applying a new kind of '<em>in situ </em> omics' technology for spatially resolved systems biology. </p>
    <p class="text_column">We use state of the art microscopy, fluidics, and Next-gen Sequencing, combined with cutting edge computational tools to analyze genomics, transcriptomics, and other information of single cells from tissues.</p>
    <p class="text_column">Such omics data can reveal molecularly defined cell types, gene networks, as well as cellular signaling events within their spatial context, all of which are critical for understanding tissue biology and disease mechanisms.</p>
  </section>
	<!-- Research Section -->
  <footer>
    <article class="footer_column">
      <h3>Biology</h3>
      <img src="images/back_old.png" alt="" width="400" height="200" class="cards"/>
      <p>Physiological processes take place under intricate cellular coordination. The 3D organization of tissue components affects gene expression and is crucial for tissue health. We are interested in studying the tumor microenvironment, especially tumor-immune interactions, whose secrets we must unlock to make immunotherapy more effective and widely applicable.</p>
    </article>
    <article class="footer_column">
      <h3>Technology</h3>
      <img src="images/lasers.png" alt="" width="400" height="200" class="cards"/>
      <p>Spatial information is often missing from common gene expression analysis methods, like next-generation and single-cell sequencing, as they require cells to be disaggregated and lysed. It was for this reason that we are developing spatial omics methods to decode the spatial organization and gene regulatory mechanisms of complex tissues.</p>
    </article>
  </footer>
  <!-- Parallax Section -->
	<section class="banner3">
    <h2 class="parallax"><a href="https://www.nature.com/articles/s41467-024-46669-y" style="color:white">Highly Sensitive Spatial Transcriptomics using FISHnCHIPs</a></h2>
    <p class="parallax_description">By simultaneously imaging ~2-35 co-expressed genes that are spatially co-localized in tissues, FISHnCHIPs derive similar spatial information as single-gene FISH, but at ~2-20-fold higher sensitivity.</p>
  </section>
	<section class="banner2">
    <h2 class="parallax"><a href="https://www.nature.com/articles/s41588-024-01664-3" style="color:white">BANKSY unifies cell typing and tissue domain segmentation for scalable spatial omics data analysis</a></h2>
    <p class="parallax_description">By constructing a product space of cell and neighbourhood transcriptomes, representing cell state and microenvironment, BANKSY improves cell type clustering and domain segmentation on spatial omics data. <a href="https://prabhakarlab.github.io/Banksy/index.html" style="color:white"> Software </a> </p>
  </section>
  <section class="banner1">
    <h2 class="parallax"><a href="https://www.nature.com/articles/s41592-020-0858-0" style="color:white">Highly Specific RNA Imaging with Split-FISH</a></h2>
    <p class="parallax_description">By using a split-probe design to achieve enhanced specificity, Split-FISH reduces off-target background fluorescence, decreases false positives, and enables accurate profiling in uncleared tissues. <a href="https://protocolsmethods.springernature.com/users/407659-jolene-goh/posts/split-fish-a-probe-design-that-enhances-the-on-target-efficiency-and-off-target-rejection-of-multiplexed-fish" style="color:white"> Read more.</a> </p>
  </section>
	<section class="banner">
    <h2 class="parallax"><a href="https://science.sciencemag.org/content/348/6233/aaa6090.full" style="color:white">Spatial Transcriptomics with MERFISH</a></h2>
    <p class="parallax_description">To image transcripts at high throughput, different species of RNAs are labeled via FISH probes in a combinatorial fashion. These Barcodes are then reconstructed via sequential rounds of hybridization and imaging.</p>
  </section>

	
 	<!-- People Section -->
	<section class="people" id="people">
        <div class="row">
            <h2>People</h2>
        </div>
		
  <div class="gallery">
<div class="thumbnail"> <a href="https://www.a-star.edu.sg/gis/our-people/faculty-staff/members/kok-hao-chen"> <span class="col-lg-4 col-md-6 col-sm-12 text-center"><img class="rounded-circle" alt="140x140" style="width: 140px; height: 140px;" src="images/kokhao.png" data-holder-rendered="true"></span>
      <h4>Kok Hao CHEN, Ph.D.</h4>
		<p>Principal Scientist II</p></a>
    </div>
	  <div class="thumbnail"> <span class="col-lg-4 col-md-6 col-sm-12 text-center"><img class="rounded-circle" alt="140x140" style="width: 140px; height: 140px;" src="images/nigel.jpg" data-holder-rendered="true"></span>
      <h4>Shiji, Nigel CHOU, Ph.D.</h4>
      <p>Senior Scientist I</p>
    </div>
     <div class="thumbnail"> <span class="col-lg-4 col-md-6 col-sm-12 text-center"><img class="rounded-circle" alt="140x140" style="width: 140px; height: 140px;" src="images/wanyi.jpg" data-holder-rendered="true"></span>
      <h4>Wan Yi SEOW</h4>
      <p>Senior Research Officer I</p>
    </div>
	   <div class="thumbnail"> <span class="col-lg-4 col-md-6 col-sm-12 text-center"><img class="rounded-circle" alt="140x140" style="width: 140px; height: 140px;" src="images/xinrui.png" data-holder-rendered="true"></span>
      <h4>Xinrui ZHOU, Ph.D.</h4>
      <p>Senior Scientist I</p>
    </div>
	   <div class="thumbnail"> <span class="col-lg-4 col-md-6 col-sm-12 text-center"><img class="rounded-circle" alt="140x140" style="width: 140px; height: 140px;" src="images/yenling.jpg" data-holder-rendered="true"></span>
      <h4>Yen Ling LEE</h4>
      <p>Lead Research Officer II</p>
    </div>
	 	   <div class="thumbnail"> <a href="https://twitter.com/Maurice_Y_Lee">  <span class="col-lg-4 col-md-6 col-sm-12 text-center"><img class="rounded-circle" alt="140x140" style="width: 140px; height: 140px;" src="images/maurice.jpeg" data-holder-rendered="true"></span>
      <h4>Maurice Youzong LEE, Ph.D.</h4>
      <p>Scientist</p></a>
    </div>
	  <div class="thumbnail"> <span class="col-lg-4 col-md-6 col-sm-12 text-center"><img class="rounded-circle" alt="140x140" style="width: 140px; height: 140px;" src="images/tu.png" data-holder-rendered="true"></span>
      <h4>Dao Tuc Tu, Ph.D.</h4>
      <p>Scientist</p>
    </div>
    <div class="thumbnail"> <span class="col-lg-4 col-md-6 col-sm-12 text-center"><img class="rounded-circle" alt="140x140" style="width: 140px; height: 140px;" src="images/caojiao.jpg" data-holder-rendered="true"></span>
      <h4>Jiao, CAO, Ph.D.</h4>
      <p>Scientist</p>
    </div>
	  <div class="thumbnail"> <span class="col-lg-4 col-md-6 col-sm-12 text-center"><img class="rounded-circle" alt="140x140" style="width: 140px; height: 140px;" src="images/jose.jpg" data-holder-rendered="true"></span>
      <h4>Jose Guijarro Nuez</h4>
      <p>A*STAR ARAP-MRC DTP Scholar (PhD student)</p>
    </div>
		</div>
	  </section>
		<div class="row">
            <h2>Former Members</h2>
        </div>
		 <section class="publications" id="alumni">
			 <p>Vipul Singhal, 2018-2024. Integrated Biosciences</p>
			 <p>Nigel Chou Shiji, 2017-2023. GIS Fellow at Genome Institute of Singapore</p>
			 <p>Lurong Wang, 2022-2024. Research Officer at Genome Institute of Singapore</p>
			 <p>Jiamin Toh, 2019-2024. Research Officer at Genome Institute of Singapore</p>
			 <p>Norbert Ha, 2019-2024. Biomedical Scientist at Delta Electronics</p>
			 <p>Jack Petts-Lewis, 2021-2023. Graduate Student at University of Warwick</p>
			 <p>Lingfan Jiang, 2019-2023. Research Scientist at KLA</p>
			 <p>Jazlynn Tan, 2022. University of Toronto</p>
			 <p>Jeeranan Boonruangkan, 2018-2022. Biofourmis</p>
			 <p>Jie Lin Jolene Goh, 2016-2022. Veranome Biosystems</p>
			 <p>Mike Jin-An Huang, 2018-2021. 1910 Genetics</p>
			 <p>Paul Cheng, 2018-2021. Veranome Biosystems</p>
			 <p>Kaviya Selva Raju, 2018-2020. Research associate at Applied Materials</p>
			 <p>Siyi Chen, 2018. Senior scientist at Maiden Therapeutics</p>
			 <p>Christabelle Goh, 2018. Scientist II at Applied Materials</p>
			 <p>Ziqing, Winston Zhao, 2018-2019. Assistant professor at National University of Singapore</p>
			 <p>Teck Por Lim, 2018-2019.</p>
			 <p>Mei Lun Chow, 2018. Project officer at Nanyang Technological University</p>
			 <p></p>
		</section>
		<div class="row">
            <h2>Publications</h2>
        </div>
		 <section class="publications" id="publications">
			 <p>Zhou, X.*, Seow, W. Y.*, Ha, N., Cheng, T. H., Jiang, L. F., Boonruangkan, J., Goh, J. J. L., Prabhakar, S., Chou, N.#, Chen, K. H.#, Highly sensitive spatial transcriptomics using FISHnCHIPs of multiple co-expressed genes, Nature Communications, (2024) DOI: 10.1038/s41467-024-46669-y.</p>
		<p>Singhal, V.*, Chou, N.*, Lee, J., Yue, Y., Liu, J., Chock, W. K., Lin, Li, Chang, Y., Teo, E., Lee, H. K., Chen, K. H.#, Prabhakar, S.#, BANKSY unifies cell typing and tissue domain segmentation for scalable spatial omics data analysis, Nature Genetics, (2024) DOI: 10.1038/s41588-024-01664-3.</p>	 
		<p>Goh, J. J. L.*, Chou, N.*, Seow, W. Y., Ha, N., Cheng, C. P. P., Chang, Y., Zhao, Z. W., Chen, K. H. Highly specific multiplexed RNA imaging in tissues with split-FISH. Nature Methods, (2020) https://doi.org/10.1038/s41592-020-0858-0.</p>	 	
	    <p>Li, H., Courtois, E. T., Sengupta, D., Tan, Y., Chen, K. H., Goh, J. J. L., et al. Reference component analysis of single-cell transcriptomes elucidates cellular heterogeneity in human colorectal tumors. Nature Genetics (49), 708–718 (2017).</p>
			 <p>Moffitt, J. R.*, Hao, J.*, Wang, G.*, Chen, K. H., Babcock, H. P. & Zhuang, X. High-throughput single-cell gene-expression profiling with multiplexed error-robust fluorescence in situ hybridization. P.N.A.S. 113 (39), 11046-11051 (2016).</p>
			 <p>Chen, K. H.*, Boettiger, A. N.*, Moffitt, J. R.*, Wang, S. & Zhuang, X. Spatially resolved, highly multiplexed RNA profiling in single cells. Science 348, aaa6090 (2015).</p>
	<p>Cha, T. G., Baker, B.A., Salgado J., Bates C.J., Chen K.H., Chang A.C., Han J.C., Strano M.S., & Choi J.H. Understanding oligonucleotide-templated nanocrystals: Growth mechanisms and surface properties. ACS Nano 6, 8136–8143 (2012).</p>
	<p>Bates, M., Dempsey, G. T., Chen, K. H. & Zhuang, X. Multicolor super-resolution fluorescence imaging via multi-parameter fluorophore detection. ChemPhysChem 13, 99–107 (2012).</p>
	<p>Chen, K. H., Hobley, J., Foo, Y. L. & Su, X. Wide-field single metal nanoparticle spectroscopy for high throughput localized surface plasmon resonance sensing. Lab Chip 11, 1895–1901 (2011).</p>
			 <p>Dempsey, G. T.*, Vaughan, J. C.*, Chen, K. H.*, Bates, M. & Zhuang, X. Evaluation of fluorophores for optimal performance in localization-based super-resolution imaging. Nature Methods 8, 1027–1036 (2011).</p>
			 <p>Ishitsuka, Y., Okumus, B., Arslan, S., Chen, K. H. & Ha, T. Temperature-independent porous nanocontainers for single-molecule fluorescence studies. Anal. Chem. 82, 9694–9701 (2010).</p>
			 <p>Mateo-Alonso, A., Ehli, C., Chen, K. H., Guldi, D. M. & Prato, M. Dispersion of single-walled carbon nanotubes with an extended diazapentacene derivative. J. Phys. Chem. A 111, 12669–12673 (2007).</p>
			 <p>Choi, J. H., Chen, K. H. & Strano, M. S. Aptamer-capped nanocrystal quantum dots: A new method for label-free protein detection. J. Am. Chem. Soc. 128, 15584–15585 (2006).</p>
  </section>
	<section class="contact" id="contact">
		
	<div class="row">
            <h2>Contact</h2>
        </div>
 	 <footer>
    <article class="footer_column">
      <p>We’re located on level 5 of the Genome Institute of Singapore, at the Biopolis research campus in the One-North area.</p>
      <p>Our mailing address is: Genome Institute of Singapore (GIS), 60 Biopolis Street Level 5, Genome Building, Singapore 138672</p>
		<p>Positions available.
Email:chenkh(at) gis.a-star.edu.sg</p>
    </article>
    <article class="footer_column">
<iframe width="90%" height="300" frameborder="0" scrolling="no" marginheight="0" marginwidth="0" src="https://www.google.com/maps?f=q&amp;source=s_q&amp;hl=en&amp;geocode=&amp;q=60 Biopolis Street, Singapore 138672&amp;ie=UTF8&amp;output=embed"></iframe>
    </article>
  </footer>
		</section>
		<!-- Copyrights Section -->
  <div class="copyright">&copy;2024 - <strong>chen lab</strong></div>
</div>
<!-- Main Container Ends -->
</body>
</html>
